runtime/lua/vim/lsp.lua

```diff
@@ -2144,11 +2145,10 @@ end
 ---@param bufnr (integer) Buffer handle, or 0 for current.
 ---@param method (string) LSP method name
 ---@param params (table|nil) Parameters to send to the server
----@param handler fun(results: table<integer, {error: lsp.ResponseError, result: any}>) (function)
+---@param handler function function parameter is results: table<integer, {error: lsp.ResponseError, result: any}>
```

Comment:
this is confusing now. isn't there a way to model a function signature?

Reply:
can build a model for it 👍

Reply:
` fun(results: table<integer, {error: lsp.ResponseError, result: any}>) (function)` already _was_ the right way to model the signature for luals. See the function type definition part from https://github.com/LuaLS/lua-language-server/wiki/Annotations#type

Please don't change stuff that's already correct

If it's too verbose, we can define an `@alias` for it, but given that it's only used once here I'm not sure that it's worth it.